app/src/main/java/com/puzzling/aoslaboratory/presentation/register/CodeCopyDialogFragment.kt

```diff
@@ -0,0 +1,68 @@
+package com.puzzling.aoslaboratory.presentation.register
+
+import android.content.ClipData
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.viewModels
+import com.puzzling.aoslaboratory.databinding.FragmentCodeCopyDialogBinding
+import com.puzzling.aoslaboratory.util.getClipboardManager
+
+class CodeCopyDialogFragment :
+    DialogFragment() {
+    private val viewModel: CodeCopyViewModel by viewModels()
+    private lateinit var clip: ClipData
+    private var _binding: FragmentCodeCopyDialogBinding? = null
+    private val binding get() = _binding!!
```

Comment:
```suggestion
    get() = requireNotNull(_binding) { "binding is null" }
```
저희 baseFragment도 !! 보다는 requireNotNull로 바꾸는건 어떨까요?!